src/System.Net.Requests/src/System/Net/FtpWebRequest.cs

```diff
@@ -212,11 +212,11 @@ public sealed class FtpWebRequest : WebRequest
 
         private bool _enableSsl;
         private FtpControlStream _connection;
-        private Stream _stream;
+        private volatile Stream _stream;
```

Comment:
Why does _stream need to be volatile?  What race condition is this preventing?

Reply:
It doesn't - will delete.